lua/lspconfig/efm.lua

```diff
@@ -8,7 +8,9 @@ local bin_name = "efm-langserver"
 configs[server_name] = {
   default_config = {
     cmd = {bin_name};
-    root_dir = util.root_pattern(".git");
+    root_dir = function(fname)
```

Comment:
You should use util.path.dirname for this

https://github.com/neovim/nvim-lspconfig/blob/f68f7d4c30abb3e4b8ebe6e3a604cc6f9eb924e8/lua/lspconfig/pyright.lua#L30-L33

Reply:
Ah, makes sense. Done.